docs/libcurl/opts/CURLOPT_FTP_SKIP_PASV_IP.3

```diff
@@ -29,14 +29,21 @@ CURLOPT_FTP_SKIP_PASV_IP \- ignore the IP address in the PASV response
 .nf
 #include <curl/curl.h>
 
-CURLcode curl_easy_setopt(CURL *handle, CURLOPT_FTP_SKIP_PASV_IP, long skip);
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_FTP_SKIP_PASV_IP, long rule);
 .fi
 .SH DESCRIPTION
-Pass a long. If \fIskip\fP is set to 1, it instructs libcurl to not use the IP
-address the server suggests in its 227-response to libcurl's PASV command when
-libcurl connects the data connection. Instead libcurl will re-use the same IP
-address it already uses for the control connection. But it will use the port
-number from the 227-response.
+URL_FTP_SKIP_PASV_IP_ALWAYS instructs libcurl to ignore the IP address the
+server suggests in its 227-response to libcurl's PASV command when libcurl connects
+the data connection. Instead libcurl will re-use the same IP address it already uses
+for the control connection. But it will use the port number from the 227-response.
+The allowed values are:
+.IP CURL_FTP_SKIP_PASV_IP_NEVER
+Always use the IP returned by the server.
+.IP CURL_FTP_SKIP_PASV_IP_ALWAYS
+Always ignore the IP returned by the server.
+.IP CURL_FTP_SKIP_PASV_IP_IF_NOT_ROUTABLE
+Only ignore the returned IP if it is within the private address space (see RFC1918),
+which is a common server configuration error.
```

Comment:
I think this should also mention from which version this option works.

Reply:
updated, but I guess @zenju has better knowledge of what he wanted to do with this 
PR